assets/js/plan_template.js

```diff
@@ -193,6 +195,22 @@ async function parseResponse(response) {
                     })
                 }
             )
+            // only shows the first 3 results
+            $('#tables .table').slice(itemsPerPage).hide();
+            $('#pagination').pagination({
+
+                // Total number of items to be paginated
+                items: totalItemsCount,
+
+                // Items allowed on a single page
+                itemsOnPage: itemsPerPage,
+                onPageClick: function (pageIdx) {
+                    const itemsOnPage = 2;
+                    $('#tables .table').hide()
+                        .slice(itemsOnPage * (pageIdx - 1),
+                            itemsOnPage + itemsOnPage * (pageIdx - 1)).show();
+                }
+            });
```

Comment:
This looks good to me. Just curious why do we need `parseResponse` to be async function? I forgot about this, is it something related ensuring the function execution order.

Reply:
Good point, I think the function does not need to be async.